app/Resources/assets/js/app.js

```diff
@@ -0,0 +1,13 @@
+var $ = require('jquery');
+window.$ = $;
+window.jQuery = $;
```

Comment:
then it means that the typeahead plugin does not support getting jQuery through CommonJS